doc/changes.md

```diff
@@ -1,5 +1,54 @@
 # Changelog
 
+## Release v0.4.1 (2021/08/07)
+
+[![Digital Object Identifier for PyGMT v0.4.1](https://zenodo.org/badge/DOI/10.5281/zenodo.5162003.svg)](https://doi.org/10.5281/zenodo.5162003)
+
+### Highlights
+
+* 🎉 **Patch release with multiple gallery examples** 🎉
+
+### Enhancements
+
+* Add common alias "verbose" (V) to grdlandmask and savefig ([#1343](https://github.com/GenericMappingTools/pygmt/pull/1343))
+
+### Bug Fixes
+
+* Change invalid input conditions in grdtrack ([#1376](https://github.com/GenericMappingTools/pygmt/pull/1376))
+* Fix bug so that x2sys_cross accepts dataframes with NaN values ([#1369](https://github.com/GenericMappingTools/pygmt/pull/1369))
+
+### Documentation
+
+* Add gallery example for grdclip ([#1396](https://github.com/GenericMappingTools/pygmt/pull/1396))
+* Add a gallery example for different colormaps in subplots ([#1394](https://github.com/GenericMappingTools/pygmt/pull/1394))
+* Add a gallery example showing individual custom symbols ([#1348](https://github.com/GenericMappingTools/pygmt/pull/1348))
+* A gallery example for the contour method ([#1387](https://github.com/GenericMappingTools/pygmt/pull/1387))
+* Add common option aliases to COMMON_OPTIONS in decorators.py ([#1407](https://github.com/GenericMappingTools/pygmt/pull/1407))
+* Add return statement to grdclip and grdgradient docstring ([#1390](https://github.com/GenericMappingTools/pygmt/pull/1390))
+* Add development toctree and PyGMT team page ([#1308](https://github.com/GenericMappingTools/pygmt/pull/1308))
```

Comment:
Maybe have this as a highlight item? Would be nice to link to https://www.pygmt.org/dev/team.html too.

Reply:
Done in 32733a6.